go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/stackrox/rox
 
-go 1.23.0
-
-toolchain go1.23.2
+go 1.23.2
```

Comment:
Was this something you changed manually? I don't think we fundamentally start to rely on any language change between .0 and .2 (as there most likely aren't any).

Reply:
I changed it manually to invalidate caches. This will be changed in next PR